packages/backend.ai-ui/src/components/BAIConfigProvider.tsx

```diff
@@ -0,0 +1,31 @@
+import { BAILocale, i18n } from '../locale';
+import { ConfigProvider, ConfigProviderProps } from 'antd';
+import { useEffect } from 'react';
+import { I18nextProvider } from 'react-i18next';
+
+export interface BAIConfigProviderProps {
+  children?: React.ReactNode;
+  locale?: BAILocale;
+  antdConfig?: Omit<ConfigProviderProps, 'locale'>;
```

Comment:
- We can extend BAIConfigProvider from ConfigProvider. I don't see any specific reason antConfig should be a  parameter
- BAIConfigProvider name is a little bit general and widely for me. BAIUIConfigProvider?